agent/src/agent/pipeline/config/jython_scripts/influx2_transform_records.py

```diff
@@ -0,0 +1,41 @@
+global sdc
+
+try:
+    sdc.importLock()
+    import sys
+    import os
+    import collections
+
+    sys.path.append(os.path.join(os.environ['SDC_DIST'], 'python-libs'))
+
+    from datetime import datetime
+finally:
+    sdc.importUnlock()
+
+DATEFORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+
+
+def to_timestamp(date):
+    epoch = datetime(1970, 1, 1)
+    return (date - epoch).total_seconds()
+
+
+records = {}
+for record in sdc.records:
+    # * 1000 because of strange implementaion of 'Add tags and static dimensions', it will divide by 1000 later
+    timestamp = to_timestamp(datetime.strptime(record.value[sdc.userParams['TIMESTAMP_COLUMN']], DATEFORMAT)) * 1000
```

Comment:
if you don't want it to be divided by 1000 then specify timestamp format as `unix` instead of `unix_ms`